remotestorage-module.js

```diff
@@ -1,4 +1,10 @@
-RemoteStorage.defineModule('myfavoritedrinks', function(privateClient) {
+// This constant will be reused by the other files
+// Enable change events for changes in the same browser window
+const remoteStorage = new RemoteStorage(
```

Comment:
As this code is not being transpiled, we can't use ES6 syntax in here. It will break in a lot of browsers that way.

Reply:
Thanks!